bot/utils/messages.py

```diff
@@ -95,8 +94,11 @@ async def wait_for_deletion(
         allow_mods=allow_mods,
     )
 
-    with contextlib.suppress(asyncio.TimeoutError):
+    try:
         await bot.instance.wait_for('reaction_add', check=check, timeout=timeout)
+    except asyncio.TimeoutError:
+        await message.clear_reactions()
```

Comment:
Perhaps we could adjust the docstring for this change?

Reply:
Good idea. [Committed](https://github.com/python-discord/bot/pull/1696/commits/aa77ce65dff915611ce5bc56d77f41b5803a5e65)

Reply:
And [re-committed.](https://github.com/python-discord/bot/pull/1696/commits/1abd1c230aedea6c0e8e680253bd7690cc27fd4c)